samples/glean/samples-glean-library/metrics.yaml

```diff
@@ -16,6 +16,7 @@ sample_metrics:
     bugs:
       - 123456789
     data_reviews:
-      - N/A 
+      - N/A
     notification_emails:
       - telemetry-client-dev@mozilla.com
+    expires: never
```

Comment:
For the samples, which people will be looking at, we should put something else other than `never` - that shouldn't be a field that is used often, so having it in the example will just lead to confusion
@mdboom

Reply:
That's a good point.  However, I'm not sure how to do that without our sample application breaking every six months...  I'll file a bug for this and come back to it, for sure.